src/main/java/ch/wisv/areafiftylan/controller/TeamRestController.java

```diff
@@ -167,17 +165,31 @@ public Team getTeamById(@PathVariable Long teamId) {
         return teamService.findTeamInvitesByTeamId(teamId);
     }
 
+    /**
+     * Accept an invite
+     *
+     * @param token Token of the Invite
+     *
+     * @return Statusmessage
+     */
     @PreAuthorize("@currentUserServiceImpl.canAcceptInvite(principal, #token)")
     @RequestMapping(method = RequestMethod.POST, value = "/invites")
     public ResponseEntity<?> acceptTeamInvite(@RequestBody String token) {
         teamService.addMemberByInvite(token);
         return createResponseEntity(HttpStatus.OK, "Invite successfully accepted");
     }
 
+    /**
+     * Decline or revoke an invite
+     *
+     * @param token Token of the Invite
+     *
+     * @return Statusmessage
+     */
     @PreAuthorize("@currentUserServiceImpl.canRevokeInvite(principal, #token)")
     @RequestMapping(method = RequestMethod.DELETE, value = "/invites")
     public ResponseEntity<?> declineTeamInvite(@RequestBody String token) {
```

Comment:
Revoke sounds as if the sender can also cancel the invite. I wouldn't name this method 'declineTeam...' but 'revokeTeam...'